### PR TITLE
Debug Windows CI failure with containerd 1.5.8

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Debug https://github.com/containerd/nerdctl/pull/533#issuecomment-972758232

### CODE REVIEW

**Comment by jsturtevant** [comment]
is there a way to retrigger a single test?  I think the changes here will work based on https://github.com/containerd/nerdctl/pull/533#issuecomment-974209675

**Comment by AkihiroSuda** [comment]
> is there a way to retrigger a single test? I think the changes here will work based on [#533 (comment)](https://github.com/containerd/nerdctl/pull/533#issuecomment-974209675)

A reviewer/committer can retrigger a single Cirrus test (and the entire GHA tests).

Would you be interested in being a reviewer (and eventually a comitter) of nerdctl ?
https://github.com/containerd/project/blob/main/GOVERNANCE.md

**Comment by jsturtevant** [comment]
As expected the exec's are working when used directly.  It seems the test framework might be holding onto a file reference somewhere:

CI run with exec directly works:
```
hello
+ echo hello2
hello2
6e6af0200fe8d0d97bea42b372263f5926f9a523c80679e9e2721e1c86f62ad8
hello3
nerdctl-test-exec
bfca23b3985de4108463b70a5c8db8b7d61192c025d23c898ba1cf87a1a74d60
+ echo hello4
hello4
```

**Comment by jsturtevant** [comment]
> A reviewer/committer can retrigger a single Cirrus test (and the entire GHA tests).
> 
> Would you be interested in being a reviewer (and eventually a comitter) of nerdctl ? https://github.com/containerd/project/blob/main/GOVERNANCE.md

Not sure I've done enough yet 😊 and I may only be helpful from Windows side given my current knowledge but would be interested.

File: .cirrus.yml [hunk 1]
```diff
@@ -19,9 +19,24 @@ docker_builder:
...
-    - go test -v -run "^(TestRunWorkdir|TestRunWithDoubleDash|TestRunExitCode|TestRunCIDFile|TestRunEnvFile|TestRunEnv|TestExec|ImageInspect|TestRunUserName)$" ./cmd/...
+
+    - nerdctl run --rm gcr.io/k8s-staging-e2e-test-images/busybox:1.29-2 echo hello
+    - nerdctl run --rm gcr.io/k8s-staging-e2e-test-images/busybox:1.29-2 sh -euxc "echo hello2"
+
+    - nerdctl run -d --name nerdctl-test-exec gcr.io/k8s-staging-e2e-test-images/busybox:1.29-2 sleep 1h
+    - nerdctl exec nerdctl-test-exec echo hello3
+    - nerdctl rm -f nerdctl-test-exec
+
+    - nerdctl run -d --name nerdctl-test-exec gcr.io/k8s-staging-e2e-test-images/busybox:1.29-2 sh -euxc "sleep 1h"
```
Comment (jsturtevant) [inline .cirrus.yml:40]:
```suggestion
    - nerdctl run -d --name nerdctl-test-exec gcr.io/k8s-staging-e2e-test-images/busybox:1.29-2 sh -eux "sleep 1h"
```


still working on why but this should let it pass

**Comment by AkihiroSuda** [comment]
> > A reviewer/committer can retrigger a single Cirrus test (and the entire GHA tests).
> > Would you be interested in being a reviewer (and eventually a comitter) of nerdctl ? https://github.com/containerd/project/blob/main/GOVERNANCE.md
> 
> Not sure I've done enough yet 😊 and I may only be helpful from Windows side given my current knowledge but would be interested.

Thanks, opened https://github.com/containerd/nerdctl/pull/562 to add you as a Reviewer

**Pull request closed without merging**